app/src/main/res/layout/fragment_first_onboarding.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+        android:id="@+id/textView"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="48dp"
```

Comment:
I suggest to create a dimens.xml file and create your standard sizes for everything you do: https://stackoverflow.com/questions/29128178/how-to-use-dimens-xml-in-android . For example, for text sizes you can have text_sm , text_md, text_base or for margin_sm, margin_md, margin_base . I usually try to follow a multiple of 4 approach for padding and margin. margin_sm = 4dp, margin_md = 8dp, margin_base = 16dp and for text I usually set text_sm  = 12sp and adjust based on the requirements.